OneCommunityGlobal/HGNRest
# Jingyi edit header message permission

# Description
Gives the user permission to edit the message displayed in the header.

## Related PRS (if any):
To test this backend PR you need to checkout the [#2345](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2345) frontend PR.
…

## Main changes explained:
- create a new initial permission for editing header message
- Add permission check before running backend logic
- 
## How to test:
1.check into current branch
2.do npm install npm run build and npm start to run this PR locally
3.Clear site data/cache
4. then Please refer to the frontend pull request
